### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by yeongsheng-tan**
git url for fullstack-tutorial does not work

### CODE REVIEW

**Comment by yeongsheng-tan** [comment]
> @yeongsheng-tan: Thank you for submitting a pull request! Before we can merge it, you'll need to sign the Meteor Contributor Agreement here: https://contribute.meteor.com/

Done.

File: docs/source/tutorial/introduction.md [hunk 1]
```diff
@@ -55,10 +55,10 @@ Now the fun begins! First, you'll need to install our developer tools:
 - [Apollo DevTools for Chrome (suggested)](https://chrome.google.com/webstore/detail/apollo-client-developer-t/jdkknkkbebbapilgoeccciglkfbmbnfm) : Our Chrome extension giving you full visibility into your client.
 - [Apollo VSCode (suggested)](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo): Our editor integration that offers intelligent autocomplete, metrics, and more.
 
-Next, in your terminal, clone [this repository](https://github.com/apollographql/fullstack-tutorial):
+Next, in your terminal, clone [this repository](https://github.com/apollographql/fullstack-tutorial.git):
```
Comment (abernix) [inline docs/source/tutorial/introduction.md]: This certainly shouldn't be necessary to be suffixed with `.git`.

File: docs/source/tutorial/introduction.md [hunk 2]
```diff
@@ -55,10 +55,10 @@ Now the fun begins! First, you'll need to install our developer tools:
...
 
 ```bash
-git clone https://github.com/apollographql/fullstack-tutorial/
+git clone https://github.com/apollographql/fullstack-tutorial.git
```
Comment (abernix) [inline docs/source/tutorial/introduction.md]: This should work either way.  If you're having problems with this, please share a screenshot of your attempt and some reproduction information.

**Comment by abernix** [comment]
These shouldn't be necessary, but would be happy to understand the problem more with additional details and a reproduction, per my above comments.

**Pull request closed without merging**